PLANET-2368 - Implement Author profile

comzeradd opened the pull request.

comzeradd commented:
PR updated with @kirdia's commit. It's ready for review.

angtheod commented on templates/tease-author.twig:
```diff
@@ -0,0 +1,27 @@
+			<span class="search-result-item-title">{{ post.get_custom_type }}</span>
+
+			{% for page_type in post.get_custom_terms %}
+				<button class="search-result-item-head no-btn" data-term_id="{{ page_type.term_id }}">{{ page_type.name }}</button>
```
there is missing js functionality for this, see [here](https://github.com/greenpeace/planet4-master-theme/blob/release-candidate/assets/js/search.js#L51)

angtheod commented on templates/tease-author.twig:
```diff
@@ -0,0 +1,27 @@
+<li id="result-row-{{ post.ID }}" class="media search-result-list-item">
+	<img class="d-flex search-result-item-image" src="{{ post.thumbnail.src('thumbnail') }}"
```
Same here, missing js funtctionality, see [here](https://github.com/greenpeace/planet4-master-theme/blob/release-candidate/assets/js/search.js#L58) and [here](https://github.com/greenpeace/planet4-master-theme/blob/release-candidate/assets/js/search.js#L63)

comzeradd commented:
@angtheod I added a commit to fix these two issues. Given that we don't have a form to submit here, like in Search, I added the filter url in a more static way instead of using Javascript.
